integration-tests/Package.swift

```diff
@@ -0,0 +1,26 @@
+// swift-tools-version: 6.0
```

Comment:
This file is very clean. Nice. Definitely an improvement over the Xcode integration tests pile of configuration files.